rdr_service/dao/bq_participant_summary_dao.py

```diff
@@ -951,7 +953,7 @@ def get_module_answers(ro_dao, module, p_id, qr_id=None):
                                 INNER JOIN questionnaire_concept qc on qr.questionnaire_id = qc.questionnaire_id
                                 INNER JOIN questionnaire q on q.questionnaire_id = qc.questionnaire_id
                         WHERE qr.participant_id = :p_id and qc.code_id in (select c1.code_id from code c1 where c1.value = :mod)
-                        ORDER BY qr.created DESC;
+                        ORDER BY qr.created;
```

Comment:
Should these be ordered by `authored` date? That way I think we'd get their latest answer, regardless of when it made it to the RDR system.

Reply:
In this case using the `created` date is the right thing.  In the RDR responses are applied to the participant summary in the order they arrive in.  To replicate that behavior, the generators need to order by `created`, not `authored`.

Reply:
Regarding the order-by: ordering by created to match how the participant summary is built makes sense if they need to match. I'll make a note to investigate further and see if that's how participant summary should function.